frontend/dashboard/components/SearchResultReposList/SearchResultReposList.tsx

```diff
@@ -14,11 +14,11 @@ export const SearchResultReposList = ({ searchValue }: SearchResultReposList) =>
   const {
     searchResults,
     isLoadingSearchResults,
-    sortModel,
     pageSize,
+    pageNumber,
     setPageNumber,
-    setSortModel,
     setPageSize,
+    onSortClick,
```

Comment:
Nitpick, but should pageSize be above `setPageSize`

Reply:
Fixed ✅